src/main/java/com/GG/T9AgeCombat/predicates/CheckLimitationPredicate.java

```diff
@@ -1,19 +1,60 @@
 package com.GG.T9AgeCombat.predicates;
 
+import com.GG.T9AgeCombat.enums.EquipmentCategoryEnum;
+import com.GG.T9AgeCombat.enums.EquipmentTypeEnum;
+import com.GG.T9AgeCombat.enums.SpecialRulePropertyEnum;
+import com.GG.T9AgeCombat.enums.UnitTypeEnum;
+import com.GG.T9AgeCombat.models.Equipment;
+import com.GG.T9AgeCombat.models.SpecialRuleProperty;
 import org.springframework.stereotype.Component;
 
+import java.util.List;
+
 import static com.GG.T9AgeCombat.common.Constants.HORDE_MODELS_PER_RANK;
+import static java.util.stream.Collectors.toList;
 
 @Component
 public class CheckLimitationPredicate {
-    private CheckLimitationPredicate() {
+
+    public CheckLimitationPredicate() {
     }
 
     public static boolean checkFirstRound(Boolean isFirstRound) {
         return isFirstRound;
     }
 
+    public static boolean checkFirstRoundNotCharging(Boolean isFirstRound, boolean isCharging) {
```

Comment:
My personal preference with boolean return methods is to name them something like `isFirstRoundNotCharging` or `hasGreatWeapon`. Minor but it better indicates to me that this is a boolean method. I don't mind if we keep them as is.